pkg/web/context.go

```diff
@@ -102,6 +103,7 @@ const (
 func (ctx *Context) HTML(status int, name string, data interface{}) {
 	ctx.Resp.Header().Set(headerContentType, contentTypeHTML)
 	ctx.Resp.WriteHeader(status)
+	fmt.Printf("%v %v %v\n", ctx.Resp, name, data)
```

Comment:
???

Reply:
see, this is why i usually debug with `panic()` 😂